kcidb/misc.py

```diff
@@ -263,7 +263,7 @@ def schema_version(string):
                 f"Invalid major version number: {string!r}"
             )
         major = int(string)
-        version = io.schema.LATEST
+        version = SCHEMA
```

Comment:
Let's keep this file similar to others and spell out `io.SCHEMA`, instead of having a direct `SCHEMA` reference.

Besides, this file is particularly low-cohesion due to its purpose (collecting bits and pieces too small for their own modules), so we have to be careful to not pollute the namespace.